cmd/titus-standalone/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Netflix/titus-executor/vk/backend"
+	corev1 "k8s.io/api/core/v1"
+
+	"encoding/json"
+	"os"
+	"time"
+
+	"github.com/Netflix/metrics-client-go/metrics"
+	"github.com/Netflix/titus-executor/config"
+	"github.com/Netflix/titus-executor/executor/runtime/docker"
+	"github.com/Netflix/titus-executor/tag"
+	"github.com/google/go-jsonnet"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/urfave/cli.v1"
+)
+
+type cliOptions struct {
+	pod      string
+	logLevel string
+}
+
+func main() {
+	var options cliOptions
+	app := cli.NewApp()
+	app.Name = "titus-standalone"
+	defer time.Sleep(1 * time.Second)
+	// avoid os.Exit as much as possible to let deferred functions run
+	cfg, cfgFlags := config.NewConfig()
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:        "pod",
+			Value:       "pod.jsonnet",
+			Destination: &options.pod,
+		},
+		cli.StringFlag{
+			Name:        "log-level",
+			Value:       "info",
+			Destination: &options.logLevel,
+		},
+	}
+	app.Flags = append(app.Flags, cfgFlags...)
+
+	dockerCfg, dockerCfgFlags := docker.NewConfig()
+	app.Flags = append(app.Flags, dockerCfgFlags...)
+
+	app.Action = func(c *cli.Context) error {
+		return cli.NewExitError(mainWithError(c, dockerCfg, cfg, &options), 1)
+	}
+
+	if err := app.Run(os.Args); err != nil {
+		panic(err)
```

Comment:
Do we have to panic?
What kind of errors do we get from this, can we just print to stderr and exit non-zero?